commands/utils/default_config.js

```diff
@@ -26,6 +26,7 @@ module.exports={
         "specs":"./*.spec.js",
         "ignore_files":"",
         "feature_file_suppport": false,
+        "network": false,
```

Comment:
We will have to handle this in the set args file to handle the defaults

Reply:
Also add it in index file to handle the params from Command line

Reply:
resolved